blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -31,8 +31,9 @@ const QueuedState = () => (
                 size: 20,
                 icon: 'timer',
                 style: { fill: '#fff' },
-            }} />
-            <span className="waiting">Waiting for run to start.</span>
```

Comment:
This is just a whitespace change. We had a peer dep error for months because the jenkins-es-lint-config plugin required ^4.0.1 and dashboard was using 5.x. You can't shrinkwrap when you have peer dep errors so the easiest fix was to roll back to 4.0.1 and make a handful of whitespace changes.